backend/core/views.py

```
@@ -1,16 +1,37 @@
 from .serializers import *
 import random
+from fsrs import FSRS, Rating, Card as f_card
+from datetime import datetime, timezone
+
+f = FSRS()
 
 # Create your views here.
-# #/ function based view
+
+class GetUser(RetrieveAPIView):
+    serializer_class = UserSerializer
```

Ruff

`UserSerializer` may be undefined, or defined from star imports

(F405)